modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/general.jsp

```diff
@@ -42,6 +42,19 @@ portletDisplay.setURLBackTitle(ParamUtil.getString(request, "backURLTitle"));
 	onSubmit="event.preventDefault();"
 	wrappedFormContent="<%= false %>"
 >
+	<aui:script sandbox="<%= true %>">
+		var groupElement = document.getElementById(
+			'<portlet:namespace />editFormInputContainer'
+		);
+
+		if (groupElement) {
+			groupElement.setAttribute(
+				'aria-label',
+				'<%= LanguageUtil.get(request, "general") %>'
+			);
+		}
+	</aui:script>
```

Comment:
Why do we set this via js? It isn't reactive here to anything, so can't we just apply this label on first render of that element?

Reply:
I was not able to find a way to get `general` in the [edit_form/start.jsp](https://github.com/liferay-frontend/liferay-portal/pull/3839/files/781a519a96b06ed8b2f4f9f71d6ab060febbac34#diff-2dcb93eaec08dfb3dc2e67cca976348b8704e5a8fef1efc30b716bf0e0cc1316R21) without hard coding it which is not ideal.
Is there a way that I might have missed, or not have yet tried?

Reply:
I was thinking it'd be an additional attribute you can add to the tag, but TBH I'm not well versed with how these tags work so maybe that is overkill

Reply:
Yeah, looking at the solution again, its quite hacky. @bryceosterhaus is right, adding a new tag attribute is the right way to go. Something like `title`, that we can set to `general` in `layout-admin-web`, and default to `portletDisplay.getTitle()` in taglib.